docs/pipelines/overview/index.md

```diff
@@ -1,15 +1,25 @@
 # What is Gruntwork Pipelines?
 
+Gruntwork Pipelines is a framework for securely deploying infrastructure as code to your AWS environments. Gruntwork Pipelines runs as a binary and series of steps in your CI system, determining what _actions_ need to be taken, in which _environments_, based on the _infrastructure changes_ that occurred.
```

Comment:
Should we mention GitOps here?

Reply:
Hm, I don't think it adds much value, but we can if you feel strongly about it.

Reply:
I don't feel strongly. I know this is a buzzword that some customers are looking for. Is it just blatantly obvious that that's what Pipelines provides and thus not worth saying?

Reply:
Added in [bea2047](https://github.com/gruntwork-io/docs/pull/1099/commits/bea2047e0a3f03c389336aa77ce4e572ab1e1f7f)